.travis.yml

```diff
@@ -17,10 +17,10 @@ install:
   - pip install flake8
 
 before_script:
-  # --exit-zero means that flake8 will not stop the build
-  # 127 characters is the width of a GitHub editor
-  - flake8 . --count --exit-zero  --max-line-length=127 --select=E999 --statistics  # syntax errors
-  - flake8 . --count --exit-zero  --max-line-length=127 --statistics
+  # stop the build if there are Python syntax errors or undefined names
```

Comment:
There are more tests enabled than this comment would imply.  Maybe just say something more generic here?